ci/docker/runtime_functions.sh

```diff
@@ -345,6 +345,25 @@ build_ubuntu_cpu_openblas() {
     report_ccache_usage
 }
 
+build_ubuntu_cpu_cmake_debug() {
+    set -ex
+    pushd .
+    cd /work/build
+    cmake \
+        -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+        -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+        -DUSE_CUDA=OFF \
+        -DUSE_MKL_IF_AVAILABLE=OFF \
+        -DUSE_OPENMP=OFF \
```

Comment:
No openmp?

Reply:
Shouldn't we test without? They were some linking issues before.

Reply:
Sure, for compilation-only that's definitely a good idea. For running the tests, do you think it makes a big difference (that needs to be validated) between enabled and disabled OpenMP?